ds001/model/model-ds000001_smdl.json

```
@@ -0,0 +1,210 @@
+  },
+  "Nodes": [
+    {
+      "Level": "run",
+      "Name": "run",
+      "GroupBy": ["run", "subject"],
+      "Transformations": {
+        "Transformer": "pybids-transforms-v1",
+        "Instructions": [
+          {
+            "Name": "Factor",
+            "Input": "trial_type"
+          },
+          {
+            "Name": "Rename",
+            "Input": [
+              "trial_type.pumps_demean",
+              "trial_type.control_pumps_demean",
+              "trial_type.cash_demean",
+              "trial_type.explode_demean"
+            ],
+            "Output": [
+              "pumps",
+              "control_pumps",
+              "cash",
+              "explode"
+            ]
+          },
+          {
+            "Name": "Scale",
+            "Input": [
+              "response_time",
+              "pumps_demean",
+              "control_pumps_demean",
+              "cash_demean",
+              "explode_demean"
+              ],
+            "Demean": false,
+            "Rescale": false,
+            "ReplaceNa": "after"
+          },
+          {
+            "Name": "Assign",
+            "Input": "response_time",
+            "Target": "pumps",
+            "Output": "pumps_rt",
+            "TargetAttr": "duration"
+          },
+          {
+            "Name": "Assign",
+            "Input": "response_time",
+            "Target": "control_pumps",
+            "Output": "control_pumps_rt",
+            "TargetAttr": "duration"
+          },
+          {
+            "Name": "Assign",
+            "Input": "response_time",
+            "Target": "cash",
+            "Output": "cash_rt",
+            "TargetAttr": "duration"
+          },
+          {
+            "Name": "Convolve",
+            "Model": "glover",
+            "Derivative": false,
+            "Input": [
+              "pumps",
+              "pumps_demean",
+              "pumps_rt",
+              "control_pumps",
+              "control_pumps_demean",
+              "control_pumps_rt",
+              "cash",
+              "cash_demean",
+              "cash_rt",
+              "explode",
+              "explode_demean"
+            ]
+          },
+          {
+            "Name": "Orthogonalize",
+            "Input": "pumps_rt",
+            "Other": "pumps"
+          },
+          {
+            "Name": "Orthogonalize",
+            "Input": "control_pumps_rt",
+            "Other": "control_pumps"
+          },
+          {
+            "Name": "Orthogonalize",
+            "Input": "cash_rt",
+            "Other": "cash"
+          }
+        ]
+      },
+      "Model": {
+        "X": [
+          "pumps",
+          "pumps_demean",
+          "pumps_rt",
+          "control_pumps",
+          "control_pumps_demean",
+          "control_pumps_rt",
+          "cash",
+          "cash_demean",
+          "cash_rt",
+          "explode",
+          "explode_demean",
+          "rot_x",
+          "rot_y",
+          "rot_z",
```

```suggestion
          "rot_?",
```

for didactic reason: to illustrate the "glob" patterns one can use in the design matrix.